src/data/processors/ccCta.ts

```diff
@@ -0,0 +1,25 @@
+import { FormattedCta } from '@/types/formatted/cta'
+import { FormattingError } from '../errors/formatting'
+import type { CCFieldCta } from '@/types/drupal/paragraph'
+
+export const ccCta = (cta?: CCFieldCta[]): FormattedCta => {
+  if (!cta?.length) {
+    return null
+  }
+  const ccItem = cta[0]
+  if (
+    !ccItem?.field_button_link?.length ||
+    !ccItem?.field_button_label?.length
+  ) {
+    throw new FormattingError(
+      'cta:link or label missing: should only be used on a Cta '
+    )
+  }
+  return {
+    id: ccItem.target_id, // or pid?
+    label: ccItem.field_button_label[0].value,
+    url:
+      ccItem.field_button_link[0]?.url?.path ||
+      ccItem.field_button_link[0]?.uri,
```

Comment:
This logic for CC CTA field_button_links having a url or uri was notorious for causing problems in content-build.

The URI might be to a node location in drupal which is an internal link, whereas a url.path is an outward facing link (if it exists). The problem is sometimes the link utility in Drupal gives editors an option to include the facility as the link and then it creates a url.path and uri option. This probably should be updated in the FieldLink structure for all links, but just made it the option here in CC CTAs.